excise_management/data/ir_model_fields.xml

```diff
@@ -0,0 +1,224 @@
+        <field name="translate" eval="True"/>
+    </record>
+    <record id="active_excise_category" model="ir.model.fields">
+        <field name="ttype">boolean</field>
+        <field name="copied" eval="True"/>
+        <field name="field_description">Active</field>
+        <field name="model_id" ref="excise_category"/>
+        <field name="name">x_active</field>
+    </record>
+    <record id="sequence_excise_category" model="ir.model.fields">
+        <field name="ttype">integer</field>
+        <field name="copied" eval="True"/>
+        <field name="field_description">Sequence</field>
+        <field name="model_id" ref="excise_category"/>
+        <field name="name">x_sequence</field>
+    </record>
+    <record id="description_excise_category" model="ir.model.fields">
+        <field name="ttype">text</field>
+        <field name="copied" eval="True"/>
+        <field name="field_description">Description</field>
+        <field name="model_id" ref="excise_category"/>
+        <field name="name">x_description</field>
+    </record>
+    <record id="sales_tax_excise_category" model="ir.model.fields">
+        <field name="ttype">many2one</field>
+        <field name="field_description">Excise Sales Tax</field>
+        <field name="model_id" ref="excise_category"/>
+        <field name="name">x_sales_tax_id</field>
+        <field name="relation">account.tax</field>
+        <field name="on_delete">cascade</field>
+    </record>
+    <record id="purchase_tax_excise_category" model="ir.model.fields">
+        <field name="ttype">many2one</field>
+        <field name="field_description">Excise Purchase Tax</field>
+        <field name="model_id" ref="excise_category"/>
+        <field name="name">x_purchase_tax_id</field>
+        <field name="relation">account.tax</field>
+        <field name="on_delete">cascade</field>
+    </record>
+    <record id="product_template_excise_category" model="ir.model.fields">
+        <field name="ttype">many2one</field>
+        <field name="copied" eval="True"/>
+        <field name="field_description">Excise category</field>
+        <field name="model_id" ref="product.model_product_template"/>
+        <field name="name">x_excise_category</field>
+        <field name="relation">x_excise_category</field>
+    </record>
+    <record id="product_template_currency_symbol" model="ir.model.fields">
+        <field name="ttype">char</field>
+        <field name="field_description">Currency symbol</field>
+        <field name="model_id" ref="product.model_product_template"/>
+        <field name="name">x_currency_symbol</field>
+        <field name="readonly" eval="True"/>
+        <field name="store" eval="False"/>
+        <field name="depends">company_id</field>
+        <field name="compute"><![CDATA[main_company = self.env['res.company']._get_main_company()
+for template in self:
+    template['x_currency_symbol'] = template.company_id.sudo().currency_id.symbol or main_company.currency_id.symbol
```

Comment:
A compute field to display the currency? Can't we user monetory widget?

Reply:
I couldn't find another way, because we absolutely need 4 digits for this field, while we want to keep the 2 digits elsewhere for monetary fields. If you know a better way, please don't hesitate to share